modules/37_industry/subsectors/sets.gms

```diff
@@ -392,202 +386,193 @@ $ifthen.cm_subsec_model_steel "%cm_subsec_model_steel%" == "processes"
+  fe2mat(all_enty,all_enty,all_te)   "Set of industry technologies to be included in en2en, which connects capex and opex to budget"
   /
-$ifthen.cm_subsec_model_steel "%cm_subsec_model_steel%" == "processes"
-    entydummy.entydummy.bf
-    entydummy.entydummy.bof
-    entydummy.entydummy.idr
-    entydummy.entydummy.eaf
-    entydummy.entydummy.bfcc
-    entydummy.entydummy.idrcc
-$endif.cm_subsec_model_steel
+    entydummy . entydummy . bf
+    entydummy . entydummy . bof
+    entydummy . entydummy . idr
+    entydummy . entydummy . eaf
+    entydummy . entydummy . bfcc
+    entydummy . entydummy . idrcc
   /
 
-secInd37_tePrc(secInd37,tePrc)   "Mapping of technologies onto industry subsectors"
+  secInd37_tePrc(secInd37,tePrc)   "Mapping of technologies onto industry subsectors"
   /
-$ifthen.cm_subsec_model_steel "%cm_subsec_model_steel%" == "processes"
     steel . idr
     steel . eaf
     steel . bf
     steel . bof
 
     steel . bfcc
     steel . idrcc
-$endif.cm_subsec_model_steel
   /
 
-fe2ppfen_no_ces_use(all_enty,all_in)   "Match ESM entyFe to ppfen that are not used in the CES tree, but for datainput for process-bases industry"
+  fe2ppfen_no_ces_use(all_enty,all_in)   "Match ESM entyFe to ppfen that are not used in the CES tree, but for datainput for process-bases industry"
   /
-$ifthen.cm_subsec_model_steel "%cm_subsec_model_steel%" == "processes"
     fesos . feso_steel
     fehos . feli_steel
     fegas . fega_steel
     feels . (feel_steel_primary, feel_steel_secondary)
-$endif.cm_subsec_model_steel
   /
+$else.cm_subsec_model_steel  !! ces implementation
+  opmoPrc   "Operation modes for technologies in process-based model"
+  /
+    standard   "Only one operation mode implemented"
+  /
+
+  !! sets only used for process-based steel left empty
+  secInd37Prc(secInd37)                        ""   / /
+  tePrc(all_te)                                ""   / /
+  mat(all_enty)                                ""   / /
+  matIn(all_enty)                              ""   / /
+  matOut(all_enty)                             ""   / /
+  matFin(mat)                                  ""   / /
+  ppfUePrc(all_in)                             ""   / /
+  route(all_te)                                ""   / /
+  ppfen_no_ces_use(all_in)                     ""   / /
+  tePrc2opmoPrc(tePrc,opmoPrc)                 ""   / /
+  tePrc2matIn(tePrc,opmoPrc,mat)               ""   / /
+  tePrc2matOut(tePrc,opmoPrc,mat)              ""   / /
+  tePrc2ue(tePrc,opmoPrc,all_in)               ""   / /
+  tePrc2teCCPrc(tePrc,opmoPrc,tePrc,opmoPrc)   ""   / /
+  tePrc2route(tePrc,opmoPrc,route)             ""   / /
+  mat2ue(mat,all_in)                           ""   / /
+  fe2mat(all_enty,all_enty,all_te)             ""   / /
+  secInd37_tePrc(secInd37,tePrc)               ""   / /
+  fe2ppfen_no_ces_use(all_enty,all_in)         ""   / /
+$endif.cm_subsec_model_steel
```

Comment:
What is that needed for? Seems hard to maintain (whoever adds a set needs to know/remember there's this list lurking down here that it needs to be added to as well...)

Reply:
Well, GAMS will help with a friendly compilation error in case something needs to be added here.  :)
Only the sets used in set calculations down below, or somewhere else in the code, are required to be here.  I just did not want to figure out one-by-one which those are.

Reply:
Ah, you got rid of all the conditions above! Please revert that! Two reasons: 
- The one on maintainability
- There will be other process-based subsectors soon. The sets are for all subsectors, the entries are subsector-specific. Your implementation will not allow switching implementations for sectors individually.